eng/pipelines/runtime-richnav.yml

```diff
@@ -0,0 +1,40 @@
+trigger:
+  branches:
+    include:
+    - master
+    - dev/infrastructure
+    - release/*.*
+
+pr: none
+
+variables:
+- template: /eng/pipelines/common/variables.yml
+# TODO: (Consolidation) Switch away from old signing/validation variables from former Core-Setup. https://github.com/dotnet/runtime/issues/1027
+- name: TeamName
+  value: dotnet-core-acquisition
+# Set the target blob feed for package publish during official and validation builds.
+- name: _DotNetArtifactsCategory
+  value: .NETCore
+- name: _DotNetValidationArtifactsCategory
+  value: .NETCoreValidation
+
+stages:
+- stage: Build
+  jobs:
+  #
+  # Checkout repository
+  #
+  - template: /eng/pipelines/common/checkout-job.yml
+
+  #
+  # Build CoreCLR for Windows
+  #
+  - template: /eng/pipelines/common/xplat-setup.yml
```

Comment:
Use `platform-matrix.yml` template instead as we do elsewhere.

Reply:
Thanks! I'm switching to that with just "Windows_NT_X64" as the target platform - is there value in adding Windows_NT_x86 or others?